workers/tests/unittests/test_ohsome_client.py

```diff
@@ -143,11 +151,162 @@ def test_build_data_with_time(self) -> None:
                 "time": str,
             }
         )
-        layer = LayerDefinitionMock()
-        data = ohsome_client.build_data_dict(layer, self.bpolys, time="2014-01-01")
+        data = ohsome_client.build_data_dict(self.layer, self.bpolys, time="2014-01-01")
         self.assertTrue(schema.is_valid(data))
-
-        layer = LayerDefinitionMock()
-        layer.ratio_filter = "mock_ratio_filter"
+        layer = LayerDefinition(
+            name="",
```

Comment:
Is a layer without a name valid? If yes, should that be the case?

Reply:
This works here because the Layer class itself has no validation built-in. Validation happens through the Pydantic request models. Passing an empty string for the layer name to the API would result in an error. Only names defined in layer_definitions.py would pass the validation.